mmengine/fileio/backends/OSS_backend.py

```diff
@@ -0,0 +1,452 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+import re
+from pathlib import Path
+from typing import Iterator, Optional, Tuple, Union
+
+import oss2  # todo delete?
+
+from .base import BaseStorageBackend
+
+
+class OSSBackend(BaseStorageBackend):
+    """OSS storage backend (for internal usage). OSSBackend supports reading
+    and writing data to multiple bucket from Endpoint. If the file path
+    contains the bucket name, OSSBackend will read data from specified bucket
+    or write data to it. Otherwise, OSSBackend will occur error.
```

Comment:
```suggestion
    """OSS storage backend for reading (writing) from (to) OSS.
    
    OSSBackend supports reading and writing data to multiple buckets
    from Endpoint. If the file path contains the bucket name,
    OSSBackend will read data from the specified buckets
    or write data to them. Otherwise, OSSBackend will occur an error.
```